src/System.Runtime/tests/System/Reflection/ReflectionTypeLoadExceptionTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit;
+
+namespace System.Reflection.Tests
+{
+    public static class ReflectionTypeLoadExceptionTests
+    {
+        [Fact]
+        public static void NullExceptionsNoNullPointerException()
+        {
+            Type[] types = new Type[1];
+            Exception[] exceptions = new Exception[1];
+            ReflectionTypeLoadException rtle = new ReflectionTypeLoadException(types, exceptions, "Null elements in Exceptions array");
+            Assert.NotNull(rtle.ToString());
+            Assert.NotNull(rtle.Message);
+            Assert.Equal(rtle.LoaderExceptions.Length, 1);
```

Comment:
The expected value should be the first parameter:

```c#
Assert.Equal(1, rtle.LoaderExceptions.Length);
```

Applies throughout this PR.

Reply:
Opened #26166 to address.